src/components/CurrencyInputPanelBridge/index.tsx

```diff
@@ -0,0 +1,301 @@
+  font-weight: ${({ selected }) => (selected ? 600 : 700)};
+  background-color: ${({ selected, theme }) => (selected ? 'transparent' : theme.primary1)};
+  border-radius: 8px;
+  height: 28px;
+  padding: ${({ selected }) => (selected ? '0' : '0 12px')};
+  color: ${({ theme }) => theme.white};
+  box-shadow: ${({ selected }) => (selected ? 'none' : '0px 6px 10px rgba(0, 0, 0, 0.075)')};
+  outline: none;
+  cursor: ${({ disableCurrencySelect }) => (disableCurrencySelect ? 'auto' : 'pointer')};
+  user-select: none;
+  border: none;
+  text-transform: uppercase;
+  letter-spacing: 0.08em;
+
+  ${({ disabled, theme }) =>
+    disabled &&
+    `
+    background-color: ${theme.purple5};
+    color: ${transparentize(0.28, theme.purpleBase)};
+    cursor: not-allowed;
+    box-shadow: none;
+    outline: none;`}
+`
+
+const LabelRow = styled.div`
+  ${({ theme }) => theme.flexRowNoWrap}
+  align-items: center;
+  color: ${({ theme }) => theme.text1};
+  font-size: 0.75rem;
+  line-height: 1rem;
+  margin-bottom: 8px;
+`
+
+const Aligner = styled.span`
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+`
+
+const StyledDropDown = styled(DropDown)<{ selected: boolean }>`
+  margin: 0 0 0 5px;
+  height: 11px;
+  width: 11px;
+  path {
+    stroke: ${({ selected, theme }) => (selected ? theme.text1 : theme.white)};
+    stroke-width: 1.5px;
+  }
+`
+
+const InputPanel = styled.div<{ hideInput?: boolean }>`
+  ${({ theme }) => theme.flexColumnNoWrap}
+  position: relative;
+  z-index: 1;
+`
+
+const Container = styled.div<{ focused: boolean }>`
+  background-color: ${({ theme }) => theme.bg1And2};
+  border: solid 1px ${({ focused, theme }) => (focused ? theme.bg3 : theme.bg1And2)};
+  border-radius: 12px;
+  transition: border 0.3s ease;
+  padding: 18px 22px;
+`
+
+const Content = styled.div``
+
+const StyledTokenName = styled.span<{ active?: boolean }>`
+  margin: ${({ active }) => (active ? '0 0 0 6px' : '0')};
+  font-size: ${({ active }) => (active ? '16px' : '11px')};
+  line-height: ${({ active }) => (active ? '20px' : '13px')};
+  letter-spacing: 0.08em;
+`
+
+const UppercaseHelper = styled.span`
+  text-transform: uppercase;
+`
+
+const FiatRow = styled.div``
+
+interface CurrencyInputPanelProps {
+  value: string
+  displayedValue: string
+  setDisplayedValue: (val: string) => void
+  onUserInput: (value: string) => void
+  onMax?: () => void
+  label?: string
+  onCurrencySelect?: (currency: Currency) => void
+  currency?: Currency | null
+  disableCurrencySelect?: boolean
+  disabled?: boolean
+  hideBalance?: boolean
+  pair?: Pair | null
+  hideInput?: boolean
+  otherCurrency?: Currency | null
+  id: string
+  showCommonBases?: boolean
+  customBalanceText?: string
+  balance?: CurrencyAmount
+  fiatValue?: CurrencyAmount | null
+  priceImpact?: Percent
+  isLoading?: boolean
+  chainIdOverride?: ChainId
+}
+
+export default function CurrencyInputPanel({
```

Comment:
Naming of this currencyInputPanel should be changed so not to be mistaken with the other one. Also could we have expanded the previous currency input panel we had since they are almost the same?

Reply:
Fixed in #908